data_steward/init_env.sh

```diff
@@ -78,5 +78,5 @@ if [ -n "${CIRCLECI}" ]; then
   echo "export UNIONED_DATASET_ID=${UNIONED_DATASET_ID}" >>"${BASH_ENV}"
   echo "export BUCKET_NAME_UNIONED_EHR=${BUCKET_NAME_UNIONED_EHR}" >>"${BASH_ENV}"
   echo "export VOCABULARY_DATASET=${VOCABULARY_DATASET}" >>"${BASH_ENV}"
-  echo "export PATH=${PATH}:${CIRCLE_WORKING_DIRECTORY}/ci" >>"${BASH_ENV}"
+  echo 'export PATH=${PATH}:${CIRCLE_WORKING_DIRECTORY}/ci' >>"${BASH_ENV}"
```

Comment:
Removing this causes failures in the integration tests.  https://app.circleci.com/pipelines/github/all-of-us/curation/3455/workflows/5d146f63-b065-4640-9142-65e42c465c6a/jobs/10395?invite=true#step-112-3.  The change from `'` to `"` was the only change in this [commit](aefa7297b2249347c83475b20b76146fe2eab63e).

Reply:
That's very odd. I think the single quote does not interpolate, so maybe that's why it doesn't raise the error? Though that would also mean the `ci` directory is not added to the path, but the scripts in `ci` are being picked up. I think the single quote is fine.

Reply:
@nishanthpp93 you are right.  It is not interpolating to the actual values.  It is instead adding that string straight to the circleci profile.  I assume the export is working because that file is referenced anew at each command.  

However, I kept looking into this.  I think this is happening because we didn't set `working_directory` for the specific command in `config.yml`.  [CIRCLE_WORKING_DIRECTORY](https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables) am going to try this and see if the double quotes works as expected for the integration tests then.

Reply:
This did not fix it.  There is circleci documentation about using `'` quotes instead of `"`.  See [this comment](https://github.com/CircleCI-Public/gcp-cli-orb/issues/22#issuecomment-715463423) in this circleci bug [question](https://github.com/CircleCI-Public/gcp-cli-orb/issues/22).  Additionally, there is some [circleci documentation](https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-shell-command) on this.  While codacy may complain, this seems to be the preferred circleci way of dealing with built-in environment variables that are used to set other variables with shells.

Reply:
@lrwb-aou: if its helpful, I updated the env setting process in my branch to this file:
https://github.com/dcarbone/curation/blob/dpc/dc_1811_circleci_vm/docker/tests/scripts/00_init_env.sh

that calls this function:
https://github.com/dcarbone/curation/blob/dpc/dc_1811_circleci_vm/docker/tests/scripts/funcs.sh#L58

May be helpful.

Reply:
@dcarbone That's nice.  I think your update should make use of it.  I don't want to break that flow.  Since `'` is a known circleci paradigm, I think it's ok to use it here until we get the dockerized build working with all it's upgrades.  I think that would be a significant amount of scope creep for this particular PR.

Reply:
I mean that to say this should probably get what you need:
```
echo PATH=\"\${PATH}:\${CIRCLE_WORKING_DIRECTORY}/ci\" >> "${BASH_ENV}"
```

See if codacy complains about that.  it shouldn't, as it doesn't complain about my PR.